downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1562 @@
+---
+layout: ballerina-release-notes
+title: Release Note
+---
+
+### Overview of Ballerina Swan Lake Alpha 
+
+This Alpha release includes the language features planned for the Ballerina Swan Lake release. Moreover, this release includes improvements and bug fixes to the compiler, runtime, standard library, and developer tooling. This release note lists only the features and updates added after the eighth preview of Ballerina Swan Lake.
+
+- [Updating Ballerina](#updating-ballerina)
+    - [For Existing Users](#for-existing-users)
+    - [For New Users](#for-new-users)
+- [Highlights](#highlights)
+- [What is new in Ballerina Swan Lake Alpha](#what-is-new-in-ballerina-swan-lake-alpha)
+    - [Packages](#packages)
+        - [Introduction of Hierarchical Package Names](#introduction-of-hierarchical-package-names)
+        - [Introduction of the Dependencies TOML file](#introduction-of-the-dependencies-toml-file)
+        - [Support to Accept an Empty Ballerina TOML File](#support-to-accept-an-empty-ballerina-toml-file)
+    - [Language](#language)
+        - [Intersection Type Support for Errors](#intersection-type-support-for-errors)
+        - [Support for Passing a Closed Record as the Rest Argument](#support-for-passing-a-closed-record-as-the-rest-argument)
+        - [Support for the Empty XML Value](#support-for-the-empty-xml-value)
+        - [Improvements to the Match Statement](#improvements-to-the-match-statement)
+        - [Support for Cyclic Union Types](#support-for-cyclic-union-types)
+        - [Updated Syntax for User-Defined Error Construction](#updated-syntax-for-user-defined-error-construction)
+        - [Changes to Casting with Errors](#changes-to-casting-with-errors)
+        - [Changes to `toString` and `toBalString` with Errors](#changes-to-toString-and-toBalString-with-errors)
```

Comment:
This sub heading link is not showing up on the MD preview.

Reply:
Fixed.